app/src/main/java/org/kiwix/kiwixmobile/webserver/StartServer.java

```diff
@@ -0,0 +1,43 @@
+package org.kiwix.kiwixmobile.webserver;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.Menu;
+import android.widget.Toast;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+import org.kiwix.kiwixmobile.R;
+import org.kiwix.kiwixmobile.zim_manager.fileselect_view.ZimFileSelectFragment;
+
+public class StartServer extends AppCompatActivity implements
```

Comment:
Rename this to `ZimHostAcitivity`.

Reply:
FIxed da0b2f3b7217d79ff2fbdacc12eada979157badf